currently env-filter isn't in Debian, and webpki-roots shouldn't be used in Debian at all

diff --git a/Cargo.toml b/Cargo.toml
index 54121eb..4f62f69 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -140,9 +140,9 @@ optional = true
 [dependencies.tracing]
 version = "0.1.30"
 
-[dependencies.webpki-roots]
-version = "0.25.0"
-optional = true
+#[dependencies.webpki-roots]
+#version = "0.25.0"
+#optional = true
 
 [dev-dependencies.futures-executor]
 version = "0.3.5"
@@ -161,7 +161,7 @@ version = "0.3"
 features = [
     "std",
     "fmt",
-    "env-filter",
+#    "env-filter",
 ]
 
 [features]
@@ -227,10 +227,10 @@ tokio-runtime = [
     "tokio/rt",
     "hickory-proto/tokio-runtime",
 ]
-webpki-roots = [
-    "dep:webpki-roots",
-    "hickory-proto/webpki-roots",
-]
+#webpki-roots = [
+#    "dep:webpki-roots",
+#    "hickory-proto/webpki-roots",
+#]
 
 #[target."cfg(windows)".dependencies.ipconfig]
 #version = "0.3.0"
